Desktop/bioconda包推送/bioconda-recipes/recipes/gcluster/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "gcluster" %}
+{% set version = "2.0.7" %}
+{% set sha256 = "190ccf3f357d153dfb05d216a9793ce035f0512a666465bdbe5479679eda0d85" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+  
+source:
+    url: https://www.microbialgenomic.cn/gz/gcluster-v{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: generic
+  run_exports:
+    - {{ pin_subpackage('pneumo-typer', max_pin="x") }}
```

Comment:
Did you use pneumo-typer recipe as a template? 

This should be your package name.

Reply:
please change pneumo-typer to gcluster, Thank you!